src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -917,9 +974,20 @@ def main():
         ),
         help="Wait for qemu to exit - not for interactive use.",
     )
+    parser.add_argument(
+        "--write-inventory",
+        default=os.environ.get("LSR_QEMU_WRITE_INVENTORY"),
+        help=(
+            "write YAML inventory to this file rather than tmp file.  "
+            "The file must be named 'inventory' or must end in '.yml'.  "
```

Comment:
Out of curiosity... If it's named not `inventory` nor does not end in `.yml`, what bad thing happens? Does it break some other tests that rely on the convention?

Reply:
> Out of curiosity... If it's named not `inventory` nor does not end in `.yml`, what bad thing happens? Does it break some other tests that rely on the convention?

I should also explain that in addition to writing this file, Ansible will also use this file at runtime, and it is at runtime that you will have the actual problems.  Ansible will not attempt to parse the file as a YAML file, and it will try all of the other inventory methods it knows - so you will get several errors like "Cannot parse as INI file", "Cannot parse as XXX format" - and it will be difficult to figure out what the actual problem is.

Reply:
Wow, it breaks the ansible execution itself... Thanks for the explanation, @richm.

Reply:
Specifically - the inventory is written here https://pagure.io/fork/rmeggins/standard-test-roles/blob/linux-system-roles/f/inventory/standard-inventory-qcow2#_664 - then standard-inventory-qcow2 will attempt to use that file with ansible here - https://pagure.io/fork/rmeggins/standard-test-roles/blob/linux-system-roles/f/inventory/standard-inventory-qcow2#_673 - and it is here that Ansible will get the strange error messages.  Of course, if you tried to later use the inventory file, you will get the same errors, but if the inventory file name is not correct, you won't get that far.